### ISSUE TITLE
J takes next line spaces

### ISSUE THREAD

**Issue by octref** [VSCodeVim/Vim#797]
### What did you do?

```
cl|ass A {
  // Two spaces in this line
}
```

Press J.

### What did you expect to happen?

```
class A {| // Two spaces after {
}
```

Although in Vim the behavior is a bit more peculiar.

```
class A {| // One spaces after {
}
```

### What happened instead?

```
  class Schema| {
}
```

### Technical details:

* VSCode Version: 1.6 Insiders
* VsCodeVim Version: 0.2.0
* OS: OS X 10.11.6

**Comment by mleech** [comment]
This looks like a duplicate of #557 and should also be fixed by #799

**Comment by xconverge** [comment]
Looks to work now with the fix @mleech did

**Comment by jpoon** [comment]
Closed with #799
